test/unit/tile.js

```diff
@@ -91,6 +91,40 @@ const assertGoogleTiles = function (directory, expectedTileSize, expectedLevels,
   });
 };
 
+// const assertIIIFTiles = function (directory, expectedSize, expectedLevels, done) {
```

Comment:
Please can you remove this commented-out test code if no longer required.

Reply:
done